tests/test_pidfile.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -
+#
+# This file is part of gunicorn released under the MIT license.
+# See the NOTICE for more information.
+
+import errno
+import sys
+
+try:
+    import unittest.mock as mock
+except ImportError:
+    import mock
+
+import gunicorn.pidfile
+
+
+def mock_patch_builtin(name):
```

Comment:
I thought we already use the latest version available of mock for Python 2. `mock_open()` has been in the stdlib version since Python 3.3 and in the PyPI version since 2012: https://github.com/testing-cabal/mock/commit/afc98d4e4eab7d66b09483fb3da1ac30e766d026 So unless I'm missing something, we are good to go without introducing a version requirement :)